### ISSUE TITLE
Folder name / config mismatch

### ISSUE THREAD

**Issue by Rich-Hall** [coldrifting/JNSQ_ScattererPatch#1]
Looks like the EVE config files specify textures in `JNSQ_Tweaks/EVE/Textures/...`, but you've named the folder inside GameData `JNSQ_ScattererPatch`, so the textures can't be found. Simple fix - either rename the folder back to `JNSQ_Tweaks`, or alter the configs. :)

P.S. Thanks for making this patch! Is the Parallax config current too?

**Comment by Krazy0** [comment]
Yes, must rename the install folder to JNSQ_Tweaks or clouds won't work.

**Comment by Unavailable14** [comment]
where do I paste this into game data or into jnsq?

**Comment by yalov** [comment]
So, CKAN `GameData/JNSQ` with all dependencies from CKAN + `GameData/JNSQ_Tweaks` which is renamed from `zip\GameData\JNSQ_ScattererPatch` — should it work?

No any `Parallax` or Paid `Volumetric Cloud` is necessary for this ? Or modification of JNSQ folder?
